.evergreen/config_generator/components/funcs/fetch_det.py

```diff
@@ -17,6 +17,17 @@ class FetchDET(Function):
             ''',
         ),
 
+        # Make shell scripts executable.
+        bash_exec(
+            command_type=EvgCommandType.SETUP,
+            working_dir="drivers-evergreen-tools",
+            script='''\
+                for file in $(find .evergreen -type f -name \*.sh); do
+                    chmod +x "$file" || exit
+                done
```

Comment:
```suggestion
                for file in $(find .evergreen -type f -name "*.sh"); do
                    chmod +rx "$file" || exit
                done
```

Minor improvements + read permission is also necessary to execute scripts.